ark/include/kernels/comm.h

```diff
@@ -100,7 +257,7 @@ DEVICE void read(int ChanId, size_t remote_offset, size_t local_offset,
     DataType *local_data = reinterpret_cast<DataType *>(local);
     DataType *remote_data = reinterpret_cast<DataType *>(remote);
     DefaultBroadcast1<InDims, InShape, DataType, OutDims, OutShape, DataType,
-                      type::Identity, true, false, UnitOutDims, NumWarps,
+                      type::Identity, false, false, UnitOutDims, NumWarps,
```

Comment:
Does this improve throughput?

Reply:
Yes, use non atomic read/write will improve the performance.